pkgs/applications/editors/emacs/elisp-packages/manual-packages/elpaca/default.nix

```
@@ -0,0 +1,30 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  git,
+  unstableGitUpdater,
+  lib,
+}:
+
+melpaBuild {
+  pname = "elpaca";
+  version = "0-unstable-2025-01-25";
+
+  src = fetchFromGitHub {
+    owner = "progfolio";
+    repo = "elpaca";
+    rev = "db2fd7258ff69fe2d100888cb8d92cf3bf94d465";
+    hash = "sha256-SseY0iU3D3cloKZy6xPp8QT0H1Cu2uGiiVG6rXq/UHg=";
+  };
+
+  nativeBuildInputs = [ git ];
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    homepage = "https://github.com/progfolio/elpaca";
+    description = "An elisp package manager";
```

```suggestion
    description = "Elisp package manager";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes